### PR TITLE
[WIP] RHCLOUD-8875: Additional unit tests

### PR DESCRIPTION
**Pull request by catastrophe-brandon**
## Link(s) to Jira
- 

## Description of Intent of Change(s)

The goal of this PR is to provide additional unit tests for the cross-account request functionality. These tests hit fundamental validation logic in the service without the overhead of creating tests in the `iqe-rbac-plugin`.

## Local Testing
How can the feature be exercised? Run the unit tests according to the normal procedure.
How can the bug be exploited and fix confirmed? N/A
Is any special local setup required? No

## Checklist
- [ ] if API spec changes are required, is the spec updated?

- [ ] are there any pre/post merge actions required? if so, document here.

- [ ] are theses changes covered by unit tests?

- [ ] if warranted, are documentation changes accounted for?

- [ ] does this require migration changes?
  - [ ] if yes, are they backwards compatible?

- [ ] is there known, direct impact to dependent teams/components?
  -  if yes, how will this be handled?

### CODE REVIEW

File: tests/api/cross_access/test_model.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    def test_request_creation_success(self):
+        """Test the creation of cross account request."""
+        self.assertEqual(self.request.target_account, "123456")
+        self.assertEqual(self.request.employee.account, "654321")
+        self.assertEqual(self.request.employee.username, "requestor_test")
+        self.assertIsNotNone(self.request.startDate)
+        self.assertEqual(self.request.endDate, self.ref_time + timedelta(10))
+        self.assertEqual(self.request.status, "requested")
+
+    def test_request_with_invalid_dates(self):
```
Comment (catastrophe-brandon) [inline tests/api/cross_access/test_model.py:62]: @astrozzc @coderbydesign Just as an example, I created an example unit test that exercises the date validation logic a little more. Normally I've been including these types of tests in the iqe-rbac-plugin, but for simpler tests like these I think they are more appropriate as unit tests.

Comment (coderbydesign) [reply tests/api/cross_access/test_model.py:62]: Yeah that makes sense to me! 🎉

Comment (astrozzc) [reply tests/api/cross_access/test_model.py:62]: Nice!!

**Comment by coderbydesign** [comment]
@catastrophe-brandon can we close this now based on the changes @astrozzc rolled into #336 ?

**Comment by coderbydesign** [comment]
@catastrophe-brandon closing based on integration from #336

**Pull request closed without merging**